lib/build/recipe/session/index.js

```diff
@@ -221,7 +221,7 @@ class SessionWrapper {
                 options,
```

Comment:
related to dynamic vs static keys, we decided to remove the useDynamicKey arg from createNewSession (so just on a global level), and to check for the kid in the JWT post session verification based on the value of the global variable (do this check if the token is >= v3). Note that if the kid is a UUID only, then we treat it as a static key.

If the setting vs the kid format doesn't match, then we throw try refresh token error.